pkg/cli/cve_cmd.go

```diff
@@ -123,6 +126,7 @@ func setupCveFlags(cveCmd *cobra.Command, variables cveFlagVariables) {
 		`zot server in USERNAME:PASSWORD format`)
 	cveCmd.Flags().StringVarP(variables.outputFormat, "output", "o", "", "Specify output format [text/json/yaml]."+
 		" JSON and YAML format return all info for CVEs")
+	cveCmd.Flags().StringVarP(variables.searchedCVE, "search", "s", "", "Search specific CVEs by name/id")
```

Comment:
I think this should be saved together with the image name and cveID in the map searchCveParams. Is that possible?
The code would be cleaner.

```suggestion
	variables.searchCveParams["searchedCVE"] = cveCmd.Flags().StringP("search", "s", "", Search specific CVEs by name/id")
```